ipalib/errors.py

```diff
@@ -1697,6 +1697,15 @@ class CertificateFormatError(CertificateError):
     format = _('Certificate format error: %(error)s')
 
 
+class CertificateMappingError(CertificateError):
```

Comment:
Could we do without adding yet another god exception? See also my other exception related comments.